### DISCUSSION TITLE
what is the intended target torch-directml version?

### DISCUSSION THREAD

**Discussion by shott92** [lshqqytiger/stable-diffusion-webui-directml#314] [General]
is the "torch-directml-0.1.13.dev221216" the intended version? it seems to be compiled for python 3.7 and pytorch 1.13
and given the dev version is from 11 months ago it just seems odd?
Just wanting to confirm that this is the version that people have and it is working with the pytorch 2.0.0 and python 3.10.x?

Thanks

**Comment by lshqqytiger** [comment]
dev branch is very old. Switch to master. master branch will work with torch 2.0.0 and torch-directml built with python 3.10/3.9. The intended version is almost always the latest one, except for Olive.
